cms/djangoapps/contentstore/views/item.py

```diff
@@ -886,7 +889,7 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
 
     xblock_info = {
         "id": unicode(xblock.location),
-        "display_name": xblock.display_name_with_default_escaped,
+        "display_name": xblock.display_name_with_default_escaped if escape_html else xblock.display_name_with_default,
```

Comment:
Why wouldn't this always be unescaped (i.e. `xblock.display_name_with_default`) and allow the template to escape how and if it needs to?